central/sensor/service/connection/worker_queue.go

```diff
@@ -66,7 +66,7 @@ func (w *workerQueue) runWorker(ctx context.Context, idx int, stopSig *concurren
 	queue := w.queues[idx]
 	for msg := queue.pullBlocking(stopSig); msg != nil; msg = queue.pullBlocking(stopSig) {
 		err := handler(ctx, msg)
-		if err != nil {
+		if err != nil && err != context.Canceled {
```

Comment:
context.Canceled can occur when Sensor disconnects

Reply:
is it worth logging that case?

Reply:
No because they can stop for any variety of reasons

Reply:
It's a lot of logspam